port_ocean/core/integrations/mixins/sync_raw.py

```diff
@@ -362,7 +362,9 @@ async def sync_raw_all(
             EventType.RESYNC,
             trigger_type=trigger_type,
         ):
-            app_config = await self.port_app_config_handler.get_port_app_config()
+            app_config = await self.port_app_config_handler.get_port_app_config(
+                use_cache=False
+            )
```

Comment:
worth adding a comment about the why, as I am not sure I understand

Reply:
I'll add a comment, this is because of if a resync is triggered due to a mappings change, it might use the cache which holds the older mappings